vuln-reach/src/javascript/lang/accesses.rs

```diff
@@ -186,19 +185,26 @@ impl<'a> AccessGraph<'a> {
                         parent.child_by_field_name(b"left").unwrap()
                     };
 
-                    if lhs.byte_range().contains(&node.start_byte()) {
-                        // Find next parent identifier if node is in LHS of the assignment.
-                        return Self::find_accessor(cursor_cache, parent);
-                    } else {
+                    if !lhs.byte_range().contains(&node.start_byte()) {
```

Comment:
This is fine since any previously visited child node works here. We just need to know if we came from lhs or rhs.